tutorials/azureml-in-a-day/azureml-in-a-day.ipynb

```diff
@@ -0,0 +1,765 @@
+    "    environment=\"aml-scikit-learn@latest\",\n",
+    "    compute=\"cpu-cluster\",\n",
+    "    experiment_name=\"train_model_credit_default_prediction\",\n",
+    "    display_name=\"credit_default_prediction\"   \n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submit the job \n",
+    "\n",
+    "It's now time to submit the job to run in AzureML. This time you'll use `create_or_update`  on `ml_client.jobs`.\n",
+    "\n",
+    "Once completed, the job will register a model in your workspace as a result of training. You can view the job in AzureML stidio by clicking on the link in the output of the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.create_or_update(job)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output of this job will look like this in the AzureML studio. You can explore the tabs for various details like metrics, outputs etc.\n",
+    "\n",
+    "![Screenshot that shows the job overview](media/job-overview.png \"Overview of the job\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy the model as an online endpoint\n",
+    "\n",
+    "Now deploy your machine learning model as a web service in the Azure cloud, an [`online endpoint`](https://docs.microsoft.com/azure/machine-learning/concept-endpoints).\n",
+    "\n",
+    "To deploy a machine learning service, you usually need:\n",
+    "\n",
+    "* The model assets (file, metadata) that you want to deploy. You've already registered these assets in your training job.\n",
+    "* Some code to run as a service. The code executes the model on a given input request. This entry script receives data submitted to a deployed web service and passes it to the model, then returns the model's response to the client. The script is specific to your model. The entry script must understand the data that the model expects and returns. When using a MLFlow model, as in this tutorial, this script is automatically created for you. Samples of scoring scripts can be found [here](https://github.com/Azure/azureml-examples/tree/sdk-preview/sdk/endpoints/online).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a new online endpoint\n",
+    "\n",
+    "Now that you have a registered model and an inference script, it's time to create your online endpoint. The endpoint name needs to be unique in the entire Azure region. For this tutorial, you'll create a unique name using [`UUID`](https://en.wikipedia.org/wiki/Universally_unique_identifier#:~:text=A%20universally%20unique%20identifier%20(UUID,%2C%20for%20practical%20purposes%2C%20unique.)."
```

Comment:
How long should the endpoint take?  Maybe add an estimate before that cell to set expectations.